From 5631f47d2c0310c4586a692c147f2f01ad554523 Mon Sep 17 00:00:00 2001
From: Kenny Kessler <kenny.kessler@motorola.com>
Date: Tue, 10 Nov 2015 15:22:09 -0600
Subject: [PATCH 467/959] IKSWM-10891: arm/dt: athene: Add USBIN phandle

Add the VADC phandle to the USB PHY so that
USBIN voltage can be displayed.

Change-Id: I01bc4b303bb90310240dca59b07ef5550584d1a5
Signed-off-by: Kenny Kessler <kenny.kessler@motorola.com>
Reviewed-on: http://gerrit.mot.com/798728
SME-Granted: SME Approvals Granted
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Jason Knopsnyder <jasonk@motorola.com>
Reviewed-by: Ravikumar Vembu <raviv@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi b/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
index 066311b4622..1545d95678d 100644
--- a/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
+++ b/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
@@ -286,6 +286,10 @@
 	};
 };
 
+&usb_otg {
+	qcom,usbin-vadc = <&pmi8950_vadc>;
+};
+
 &pmi8950_charger {
 	status = "okay";
 	/delete-property/ qcom,iterm-ma;
-- 
2.11.0

